docassemble/assemblylinewizard/data/questions/feedback.yml

```diff
@@ -0,0 +1,515 @@
+---
+metadata:
+  title: Assembly Line Interview Generator Feedback Form
+  short title: Feedback
+---
+modules:
+  - docassemble.GithubFeedbackForm.github_issue
+---
+# This is the repository that Github issue will be created on if
+# the repository is not passed as a URL argument
+code: default_repository = 'docassemble-assemblylinewizard'  
+---
+code: default_github_user_or_organization = 'suffolklitlab'
+---
+comment: |
+  For security reasons, you should limit
+  the ability of this script to create issues to any repositories
+  that you control.
```

Comment:
Does this mean the same thing as "this script should only be allowed to create issues on repositories that you control"?